pages/common/rename.md

```diff
@@ -10,7 +10,7 @@
 
 `rename -n {{'s/foo/bar/'}} {{\*}}`
 
-- Convert filenames to lower case:
+- Convert filenames to lower case (use `-f` in case-insensitive filesystems to prevent "already exists" errors):
```

Comment:
Just wondering - does "use -f to overwrite existing files" make sense ? Its just that reading the entire "in case-insensitive filesystems to prevent "already exists" errors" is trying to suggest that adding the '-f' flag prevents some sort of errors. Whereas the previous description says what its actually used for.